### DISCUSSION TITLE
Can you migrate up but allow failures?

### DISCUSSION THREAD

**Discussion by bnussman** [mikro-orm/mikro-orm#3978] [Q&A] [answered]
I really want the ability to run `mikro-orm migration:up` (or `await migrator.up();`) but the command fails if it tries to run a migration that has already been done to the database. **Is there any way possible to force `mikro-orm migration:up` to run all migrations in order even if some of them error?**

![Screen Shot 2023-01-28 at 12 40 49 PM](https://user-images.githubusercontent.com/6440455/215282361-729dcbfc-c760-40c0-9f9b-506623a42f4f.jpg)
> This error makes sense because this the database has already been initialized via `mikro-orm schema:create --run` but I still want to run old and new migrations on my local db for simplicity in local development

**Comment by B4nan** [comment, marked as answer]
Nope not possible and I would say a non-goal. The real question here is how come you have an out-of-sync migrations table. As otherwise, this would never happen.

> This error makes sense because this the database has already been initialized via mikro-orm schema:create --run but I still want to run old and new migrations on my local db for simplicity in local development

You need initial migration if you did that. Drop all existing migrations you have and generate the initial one, it will be marked as executed automatically if you schema is already in sync with entity definition.

**Reply by bnussman** [reply]
Ah, ok, this is very helpful. Thank you @B4nan. You 🪨
